src/cml.js

```diff
@@ -477,7 +478,7 @@ class CML {
         await exec('git fetch --unshallow');
```

Comment:
With your default value, I believe we need to add a return to halt termination. 
```suggestion
        return await exec('git fetch --unshallow');
```